lms/templates/instructor/instructor_dashboard_2/membership.html

```diff
@@ -17,140 +17,141 @@
         <thead>
           <tr>
             {{#labels}}
-              <td class="label">{{.}}</td>
+              <th class="label" scope="column">{{.}}</th>
             {{/labels}}
           </tr>
         </thead>
         <tbody></tbody>
       </table>
     </div>
     <div class="bottom-bar">
-      <label for="add-field" class="sr">{{add_placeholder}}</label>
-      <input type="text" id="add-field" name="add-field" class="add-field" placeholder="{{add_placeholder}}">
+      <label>
+          <span class="label-text sr">{{add_placeholder}}</span>
+          <input type="text" id="add-field" name="add-field" class="add-field" placeholder="{{add_placeholder}}">
+      </label>
       <input type="button" name="add" class="add" value="{{add_btn_label}}">
     </div>
   </div>
 </script>
 
-<div class="batch-enrollment membership-section">
-  <h2> ${_("Batch Enrollment")} </h2>
-  <p>
-    <label for="student-ids">
-      ${_("Enter email addresses and/or usernames separated by new lines or commas.")}
-      ${_("You will not get notification for emails that bounce, so please double-check spelling.")} </label>
-    <textarea rows="6" name="student-ids" placeholder="${_("Email Addresses/Usernames")}" spellcheck="false"></textarea>
-  </p>
-  <input type="hidden" id="is_course_white_label" value="${section_data['is_white_label']}">
-  % if section_data['is_white_label']:
-  <p>
-    <label for="reason-field-id">
-      ${_("Enter the reason why the students are to be manually enrolled or unenrolled.")}
-      ${_("This cannot be left blank and will be recorded and presented in Enrollment Reports.")}
-      ${_("Therefore, please give enough detail to account for this action.")} </label>
-    <textarea rows="2" id="reason-field-id" name="reason-field" placeholder="${_('Reason')}" spellcheck="false"></textarea>
-  </p>
-  %endif
-  <div class="enroll-option">
-    <input type="checkbox" name="auto-enroll" value="Auto-Enroll" checked="yes">
-    <label for="auto-enroll">${_("Auto Enroll")}</label>
-    <div class="hint auto-enroll-hint">
-      <span class="hint-caret"></span>
-      <p>
-	${_("If this option is <em>checked</em>, users who have not yet registered for {platform_name} will be automatically enrolled.").format(platform_name=settings.PLATFORM_NAME)}
-	${_("If this option is left <em>unchecked</em>, users who have not yet registered for {platform_name} will not be enrolled, but will be allowed to enroll once they make an account.").format(platform_name=settings.PLATFORM_NAME)}
-	<br /><br />
-	${_("Checking this box has no effect if 'Unenroll' is selected.")}
-      </p>
+<h2 class="hd hd-2">Enrollment</h2>
```

Comment:
I added a heading because:

* I updated the `h2`'s in each section to `legend`'s
* provides structural context for page layout